change/@internal-react-components-27920e95-e44e-4cc2-91a2-50484465e1ec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Added strings for sendBox component",
```

Comment:
nit: would be useful for the changelog to mention here now the send box steals focus when mounted

Reply:
yup can add that!